frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx

```diff
@@ -271,7 +268,7 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
       .then(() => {
         toast.success('User removed from the group');
       })
-      .catch(({ error }) => {
+      .catch(({ error, data }) => {
```

Comment:
I can see this data inside above trycatch lines. if we are not using this. shall we remove.